eo-maven-plugin/src/main/java/org/eolang/maven/HashOfTag.java

```diff
@@ -56,19 +63,31 @@ final class HashOfTag {
      */
     public String hash() throws IOException {
```

Comment:
@EugeneDar maybe it's worth mentioning in JavaDoc for this method that it supports concurrency.

Reply:
@mximp I will add it.